package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "error-catch-notifier",
+  "version": "1.0.0",
+  "description": "Helpers for catching and reporting errors",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run lint && npm run test &&  npm run compile",
+    "compile": "babel src/error-catch-notifier.js --out-file dist/error-catch-notifier.js && babel src/index.js --out-file dist/index.js",
```

Comment:
I feel like I can make this less verbose but I don't know anything about bash or how to target directories with `babel`